Riot/AppDelegate.m

```diff
@@ -1176,6 +1276,54 @@ - (void)application:(UIApplication *)application handleActionWithIdentifier:(NSS
     completionHandler();
 }
 
+// iOS 10+, see application:didReceiveLocalNotification:
```

Comment:
In fact this method is not equivalent to `application:didReceiveLocalNotification`, when the user tap on a local notification with `UILocalNotification` it calls at the end `application:didReceiveLocalNotification`. But with `UserNotifications` it calls `-[UNUserNotificationCenterDelegate userNotificationCenter:didReceiveNotificationResponse:withCompletionHandler:]`

Reply:
https://github.com/vector-im/riot-ios/pull/2207/commits/1c90e0cb4427b76c19457d8df2c3341bc01a031e

Reply:
https://github.com/vector-im/riot-ios/pull/2207/commits/7ae3070e0ae39b0ff50c33f11ac7597e8e3ad50c